games-engines/scummvm-tools/scummvm-tools-2.7.0.ebuild

```diff
@@ -0,0 +1,67 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+WX_GTK_VER=3.0-gtk3
+inherit wxwidgets toolchain-funcs
+
+DESCRIPTION="Utilities for the SCUMM game engine"
+HOMEPAGE="https://www.scummvm.org/"
+SRC_URI="https://www.scummvm.org/frs/scummvm-tools/${PV}/${P}.tar.xz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~ppc64 ~x86"
+IUSE="flac iconv mad png vorbis"
+RESTRICT="test" # some tests require external files
+
+RDEPEND="
+	dev-libs/boost:=
+	sys-libs/zlib
+	x11-libs/wxGTK:${WX_GTK_VER}
+	flac? ( media-libs/flac:= )
+	iconv? ( virtual/libiconv media-libs/freetype:2 )
+	mad? ( media-libs/libmad )
+	png? ( media-libs/libpng:= )
+	vorbis? ( media-libs/libvorbis )
+"
+DEPEND="
+	${RDEPEND}
+"
+BDEPEND="
+	virtual/pkgconfig
+"
+
+PATCHES=(
+	"${FILESDIR}/${PN}-2.7.0-guisuffix.patch"
+	"${FILESDIR}/${PN}-2.7.0-binprefix.patch"
+)
+
+src_prepare() {
+	default
+
+	rm -r *.bat dists/win32 || die
```

Comment:
Do you actually *need* to remove these, i.e. do they get installed or sth?

Reply:
Inherited from existing ebuild. (The only change in the ebuild is the patches and the license which was from GPL2 to GPL3.)